paper/paper.md

```diff
@@ -19,22 +19,22 @@ bibliography: paper.bib
 
-The generator methodology approaches are:
+The generator methods are:
 
   - Context Free Grammar [@van1996generalized]
   - Resource Description Framework (RDF, triple store) [@faye2012survey]
   - Finite State Machine [@maletti2017survey]
   - Template Strings [@copestake1996applying]
 
-Each method offers different options to the user. With the Finite State Machine approach, the function *generate_fsm_corpus* allows for users to produce a deterministic set of productions so that upon successful training accuracies of 100% can be achieved. An application of this is that different models can be tested on the amount of computational requirements they have for a particular value of complexity. The 4 approaches cover a wide range of text expansion production methods that does not have parameter estimations as part of their usage requirements. The letters of the alphabet begin in the *HANGUL* start (44032, 0xAC00) and does not apply repetition of labels with integer identifiers at the end which are only convenient for implementations but may not be ideal for some tokenization approaches. The reason for this choice is that Hangul Syllables block is one of the largest continuous blocks in the Unicode standard. It contains over 11,000 characters without interruption allowing for large alphabet and punctuation subsets to be selected.
+Each method offers different options to the user. With the Finite State Machine approach, the function *generate_fsm_corpus* produces a deterministic set of corpora so that upon successful training accuracies of 100% can be achieved. This can be used, e.g., to test the computational requirements for various models, for a particular value of complexity. The four approaches listed above, cover a wide range of text expansion production methods that does not have parameter estimations as part of their usage requirements. The letters of the alphabet begin in the *HANGUL* start (44032, 0xAC00) and does not apply repetition of labels with integer identifiers at the end which are only convenient for implementations but may not be ideal for some tokenization approaches. The reason for this choice is that Hangul Syllables block is one of the largest continuous blocks in the Unicode standard. It contains over 11,000 characters without interruption allowing for large alphabet and punctuation subsets to be selected.
```

Comment:
This sentence needs to be re-written: it's too long and I can't understand that.

    The letters of the alphabet begin in the HANGUL start (44032, 0xAC00) and does not apply repetition of labels with integer identifiers at the end which are only convenient for implementations but may not be ideal for some tokenization approaches.